### ISSUE TITLE
Field partner in statement

### ISSUE THREAD

**Issue by ThijsvOers** [onesteinbv/addons-container#65]
To be discussed next week, I noticed that the partner of the bank statement line is not filled, this makes the reconcile process more difficult. It would be great if we could add the partner info (the partner is available in the EE ponto connection)

**Comment by ThijsvOers** [comment]
@astirpe I have added 2 screenshots, maybe you can already see where the partner info is coming from and where we want to see it 
![myponto EE](https://github.com/onesteinbv/addons-container/assets/30256071/8f806e60-d589-4472-9e51-2945a68666d1)
![myponto CE](https://github.com/onesteinbv/addons-container/assets/30256071/88a2d949-00c8-4a97-a5ff-fbdad4ac3c56)

**Comment by ThijsvOers** [comment]
@astirpe I think the field is: counterpart name: 
![image](https://github.com/onesteinbv/addons-container/assets/30256071/a05144d7-66ab-40a2-b561-4b83a805450c)

**Comment by ThijsvOers** [comment]
@astirpe is this task clear for you or do you want to discuss it online? FYI: I will be at Onestein on Wednesday.

**Comment by astirpe** [comment]
@ThijsvOers is clear for me and I'm working on it, thanks

**Comment by astirpe** [comment]
Should be fixed in https://github.com/onesteinbv/addons-oca/commit/e3dd5b7ce24a835db2088f72449ec96c7579c392

**Comment by ThijsvOers** [comment]
@astirpe I have tested the changes and I do see for some records the partner info. However there are also records, where I would expect a name and it's not shown. In my example you wil see as partner 'Restaurant Kadens' and this partner is not in my contact database. If we do not fill this info, I have no clue how to reconcile the line.

It looks like in the EE edition, this field is always filled with the Counterpartname info, even in the case there is no contact in the database, I have just tested this in my own Online environment and there I can see the partner info, but the contact is not created when I reconcile.

So in my view the best way forward is to always show the Counterpartname in the Partner field without checking if the contact is in the database.
